visualization/app/codeCharta/util/fileValidator.spec.ts

```diff
@@ -179,15 +179,14 @@ describe("FileValidator", () => {
 
 		beforeEach(() => {
 			file = clone(fileWithFixedFolders)
-			folder1 = file.nodes[0].children[0]
-			folder2 = file.nodes[0].children[1]
+			;[folder1, folder2] = file.nodes[0].children
```

Comment:
move semicolon behind `file = clone(fileWithFixedFolders)` maybe?

Reply:
That's done by prettier.